plugin/parser/differ/mysql/differ.go

```diff
@@ -6,25 +6,88 @@ import (
 
 	"github.com/pingcap/tidb/parser/ast"
 	"github.com/pingcap/tidb/parser/format"
+	"github.com/pingcap/tidb/parser/model"
 )
 
+// columnMap maps column name to columnDef.
+type columnMap map[string]*ast.ColumnDef
```

Comment:
```
type columnKey struct {
  tableName string
  columnName string
}

var columnMap map[columnKey]*ast.ColumnDef
```

This can simplify things.

Reply:
PTAL.